.github/workflows/publish_en.yml

```diff
@@ -35,9 +35,18 @@ jobs:
         with:
           cache-version: 1
 
-      - name: Render and Publish EN
-        uses: quarto-dev/quarto-actions/publish@v2
-        with:
-          target: gh-pages
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Render book
+        run: Rscript -e 'babelquarto::render_book()'
```

Comment:
Should this not be `babelquarto::render_website()`?

Reply:
yes!!! sorry